docs/getting-started/package-infrastructure.md

```diff
@@ -58,197 +58,6 @@ into a package.
 First we'll create a `CompositeResourceDefinition` (XRD) to define the schema of
 our `CompositePostgreSQLInstance` and its `PostgreSQLInstance` resource claim.
 
-<ul class="nav nav-tabs">
```

Comment:
Nice catch 👍 This shows off composition better with the tabs removed for the definition!

Reply:
Duplicating this `definition.yaml` file was actually intentional. It allows each `doc/snippets/package` subdirectory to be a valid package that includes everything it needs (definition, composition, and meta). This makes it quick and easy for us to publish the "getting-started" packages used in this guide.

Perhaps we could make the definition in each package directory a symlink to this single definition, to get the best of both worlds?

Reply:
Can we just keep all the existing ones plus the new one at `https://raw.githubusercontent.com/crossplane/crossplane/master/docs/snippets/package/definition.yaml`?

Reply:
I'd be okay with that too. My thinking is that iirc Git supports symlinks, and that linking them all to one "source" file (probably the new one muvaf created) could help avoid them diverging.

Reply:
@negz Symlinking is supported by Git but I was suspicious whether `kubectl crossplane build configuration` supported it and from my experiment, it seems that symlinks are skipped but hardlinks are OK. So, I just went for hardlink.

Reply:
Well, hardlinks aren't supported by Git since it doesn't save node information of files.

Reply:
@muvaf yeah I feel like just maintaining multiple copies may be best option here...